SSO Service Name Migration community notification (#608)
BCDevOps/developer-experience: docs/kcMigration/sso-service-name-migration-notification.md

### What is happening?

> Hi teams! We have successfully introduced the new SSO service name at `oidc.gov.bc.ca`! The next step is to start migrating your application SSO integration to the new name. Please note that this does not affect the service offering but just to change how you are accessing it. For more information on this Single Sign-On service, refer to [here](https://developer.gov.bc.ca/BC-Government-SSO-Service-Definition).

Reviewer: ```suggestion
Hi teams! We have successfully introduced the new SSO service name at `oidc.gov.bc.ca`! The next step is to start migrating your application SSO integration to the new name. Please note that this change did not affect the service offering (no changes to the functionality) and only changes how the SSO service is accessed by applications. For more information on this Single Sign-On service, refer to [here](https://developer.gov.bc.ca/BC-Government-SSO-Service-Definition).
```

<!-- TODO: update sso wiki, sso service definition, and other reference! -->

For your reference, here are the original SSO service name/endpoint:
- sso-dev.pathfinder.gov.bc.ca
- sso-test.pathfinder.gov.bc.ca
- sso.pathfinder.gov.bc.ca

> And the new Service name/endpoint to migration to:

Reviewer: ```suggestion
And the new Service name/endpoint to migrate to:
```

- dev.oidc.gov.bc.ca
- test.oidc.gov.bc.ca
- oidc.gov.bc.ca

### When?

> Starting on Oct 6th, all dev/test/prod KeyCloak instances are accessible via the new service name. Product team should schedule on follow up actions and start migration.

Reviewer: ```suggestion
Starting on Oct 6th, all dev/test/prod KeyCloak instances are accessible via the new service name. Product team are encouraged to start planning the migration to the new SSO service name over the next 6 weeks.
```

> **Please note that we are not shutting down the original service name at `sso.pathfinder.gov.bc.ca`. Instead, both pathfinder and the new `oidc.gov.bc.ca` service name will be supported for four weeks/two sprints (will be extended longer if needed), which allows product teams to update integrated applications and conduct thorough testing.** As a result, we are expecting to deprecate the pathfinder service name for the following timeline:

Reviewer: ```suggestion
**Please note that we are not shutting down the original service name at `sso.pathfinder.gov.bc.ca`. Instead, both pathfinder and the new `oidc.gov.bc.ca` service name will be supported for 6 weeks/two sprints (will be extended longer if needed), which allows product teams to update integrated applications and conduct thorough testing.** As a result, we are expecting to deprecate the pathfinder service name for the following timeline:
```
